tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -4,47 +4,133 @@
 // </copyright>
 
 using System;
+using System.IO;
 using System.Net;
+using System.Text;
 
-using Datadog.Trace.Logging;
-using Datadog.Trace.Util;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
 
 namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS
 {
     internal class LambdaCommon
     {
-        private const string TraceContextEndpointEnvName = "_DD_TRACE_CONTEXT_ENDPOINT";
-        private const string TraceContextEndpoint = "http://127.0.0.1:8124/lambda/trace-context";
         private const string PlaceholderServiceName = "placeholder-service";
         private const string PlaceholderOperationName = "placeholder-operation";
+        private static readonly string DefaultJson = "{}";
```

Comment:
This is a compile-time constant, so it can be `const`.
```suggestion
        private const string DefaultJson = "{}";
```

Reply:
✅